src/components/DropMenu/DropMenu.tsx

```diff
@@ -108,14 +111,14 @@ OptionGroup.defaultProps = {
 	isHidden: false,
 };
 
-interface IDropMenuOptionProps extends StandardProps {
-	description?: string;
+export interface IDropMenuOptionProps extends StandardProps {
 	isDisabled?: boolean;
 	isHidden?: boolean;
 	isWrapped?: boolean;
 }
 
-const Option = (_props: IDropMenuOptionProps): null => null;
+const Option: FC<IDropMenuOptionProps> = (_props: IDropMenuOptionProps): null =>
```

Comment:
I might be reading between the lines but it seems like you needed to stop relying on type inference because you needed the types elsewhere. A couple questions if that's the case:

1. would simply using `typeof` work better instead of `FC`? The only issues I could see with the explicit `FC` type would be related to default props but I don't think any of these child components are using default props
1. if we do stick with the explicit `FC` type then I think we can remove the `_props: IDropMenuOptionProps` for example since that was only there to aid type inference

Reply:
I think this may just be a remnant of us trying to debug a type error involving the Option child component. I removed all the uses of FC in DropMenu and it looks like we can get away without it.